core/quantity/src/Interfaces.ts

```diff
@@ -62,6 +62,7 @@ export interface UnitConversionSpec {
 export interface UnitConversion {
   factor: number;
   offset: number;
+  error: boolean;
```

Comment:
Is this supposed to portray an error status?

Reply:
not real familiar with this, but the FormatterSpec from the quantity package is setting this with a boolean if there is no input unit set (line 74).

Reply:
@christophermlawson - this `error` boolean and its use in the FormatterSpec should not exist.  Neither are used in code on the master branch.

Reply:
Basically all changes in FormatterSpec.ts should be reverted.

Reply:
I've remove the prop and updated FormatterSpec

Reply:
Did you push, because I don't see any changes yet?

Reply:
sorry - should be there now.